cinema_service/settings.py

```diff
@@ -133,8 +141,9 @@
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = "/files/media"
```

Comment:
The MEDIA_ROOT setting is correctly configured to use a specific directory for media files. This is important for handling media files in a Docker environment.